lib/mi_bridges/driver/job_income_more_about_page.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class JobIncomeMoreAboutPage < BasePage
+      TITLE = /More About .*'s Job/
+
+      delegate :members, to: :snap_application
+
+      def setup; end
+
+      def fill_in_required_fields
+        fill_in "Name of Employer", with: current_member.employed_employer_name
+        select current_member.employed_pay_interval,
```

Comment:
are all of the payment intervals that we have in MI Bridges as well?

Reply:
They are now :)